DataFormats/PatCandidates/interface/TauPFEssential.h

```diff
@@ -48,6 +48,12 @@ struct TauPFEssential {
   reco::VertexRef sv_;
   Point svPos_;
   CovMatrix svCov_;
+  float ip3d_;
+  float ip3d_error_;
+  float ip3d_Sig_;
```

Comment:
This is what this pull request is about. These variables are filled in PATTauProducer.cc. This information will be used for calculating the MVA isolation discriminator based on miniAOD information. Also see initiating comment to this PR.